config/clients/java/template/Java/auth/HttpBasicAuth.mustache

```diff
@@ -0,0 +1,42 @@
+{{>licenseInfo}}
```

Comment:
We don't support basic auth, so this file can be dropped

Reply:
Removed in [6f9804d](https://github.com/openfga/sdk-generator/pull/158/commits/6f9804d21724edeff04096b82f2a999ab7a75144)